src/pg/sql/getArgSql.js

```diff
@@ -72,11 +72,14 @@ export default function getArgSql(
     $order &&
     getJsonBuildTrusted({ $order: sql`${JSON.stringify($order)}::jsonb` });
 
-  const cursorKey = getJsonBuildTrusted({
-    $cursor: sql`jsonb_build_array(${join(groupCols || orderCols)})`,
-  });
+  const cursorKey =
+    $group !== true &&
+    getJsonBuildTrusted({
+      $cursor: sql`jsonb_build_array(${join(groupCols || orderCols)})`,
+    });
 
-  key = sql`(${join([key, orderKey, cursorKey].filter(Boolean), ` || `)})`;
+  const keys = [filterKey, orderKey, cursorKey].filter(Boolean);
+  const key = keys.length > 0 ? sql`(${join(keys, ` || `)})` : undefined;
```

Comment:
This is great, but I think there is a further improvement we can make. We can move groupKey here, and do:

```suggestion
  const groupKey =
    Array.isArray($group) &&
    getJsonBuildTrusted({ $group: sql`${JSON.stringify($group)}::jsonb` });
    
  const key = sql`${join([
    filterKey, groupKey, orderKey, cursorKey
  ].filter(Boolean), ` || `)}`;
```

Then, getAggMeta can be simplified to drop the $group argument, and just have:
```ts
export const getAggMeta = (key, { verDefault }) =>
  getJsonBuildTrusted({
    $key: key,
    $ver: raw(verDefault),
  });
```

Reply:
Please verify that `getAggMeta` is not used anywhere else, before making this change.